village/src/components/Smurfs.js

```diff
@@ -1,10 +1,22 @@
 import React, { Component } from 'react';
-
 import Smurf from './Smurf';
+import axios from 'axios';
 
 class Smurfs extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
-
+  constructor(props) {
+    super(props);
+    this.setState = {
+      smurfs: [],
+    }
+  }
+  componentDidMount() {
+    axios.get(`http://localhost:3333/smurfs`)
+    .then(response => {
+      this.setState({ smurfs: response.data });
+    }).catch(error => {
+    });
+  }
   render() {
     return (
       <div className="Smurfs">
```

Comment:
If you change this to a functional component and keep the array of smurfs in state on the App.js file/component, you'll be able to fix the rendering issue when you make an ajax post request.